README.md

```diff
@@ -1 +1,16 @@
 # hw01
+
+## Introduction
+
+My name is **Diqi Zeng**. I am a MAPSS student concentrating in psychology. I am interested in developmental psychology. Specifically, I am interested in children's language development. I am currently working in *Dr. Amanda Woodward's lab*, studying children's different attitudes towards various language speakers. I am very excited to attend this course and hope to gain more knowledge about R to help analyze my data in the near future!
+
+### A Little More about Me
+
+My favorite foods includes:
+1. Curry 
+2. Sushi
+3. Tofu
+
+## Photography
+
+![Diqi's photo](/IMG_1576.JPG)
```

Comment:
Since the file "/IMG_1576.JPG" refers to a local path, other people would not be able to see the photo if they only download the readme.md file. I see that the rendering is fine because now the md file and jpg file are in the same folder in the repo. But the path, https://github.com/DiqiZeng/hw01/blob/master/IMG_1576.JPG?raw=true is the URL address for the uploaded photo file.